executor/admin_test.go

```diff
@@ -1322,144 +1322,12 @@ func TestCheckFailReport(t *testing.T) {
 	}()
 }
 
-func (s *testSuiteJoinSerial) TestAdminCheckTableFailed(t *testing.T) {
-	tk := testkit.NewTestKit(t, s.store)
-	tk.MustExec("use test")
-	tk.MustExec("drop table if exists admin_test")
-	tk.MustExec("create table admin_test (c1 int, c2 int, c3 varchar(255) default '1', primary key(c1), key(c3), unique key(c2), key(c2, c3))")
-	tk.MustExec("insert admin_test (c1, c2, c3) values (-10, -20, 'y'), (-1, -10, 'z'), (1, 11, 'a'), (2, 12, 'b'), (5, 15, 'c'), (10, 20, 'd'), (20, 30, 'e')")
-
-	// Make some corrupted index. Build the index information.
-	s.ctx = mock.NewContext()
```

Comment:
Are these cases moved somewhere else?

Reply:
Yep. The changes are in #30310 which should can be merged first.